options.go

```diff
@@ -0,0 +1,67 @@
+package allure
+
+import (
+	"github.com/dailymotion/allure-go/severity"
+)
+
+type Option func(r hasOptions)
+
+func Lead(lead string) Option {
+	return func(r hasOptions) {
+		r.addLabel("lead", lead)
+	}
+}
+
+func Owner(owner string) Option {
+	return func(r hasOptions) {
+		r.addLabel("owner", owner)
+	}
+}
+
+func Epic(epic string) Option {
+	return func(r hasOptions) {
+		r.addLabel("epic", epic)
+	}
+}
+
+func Severity(severity severity.Severity) Option {
+	return func(r hasOptions) {
+		r.addLabel("severity", string(severity))
+	}
+}
+
+func Story(story string) Option {
+	return func(r hasOptions) {
+		r.addLabel("story", story)
+	}
+}
+
+func Feature(feature string) Option {
+	return func(r hasOptions) {
+		r.addLabel("feature", feature)
+	}
+}
+
+func Parameter(name string, value interface{}) Option {
+	return func(r hasOptions) {
+		r.addParameter(name, value)
+	}
+}
+
+func Description(description string) Option {
+	return func(r hasOptions) {
+		r.addDescription(description)
+	}
+}
+
+func Body(action func()) Option {
```

Comment:
Could we rename it into Action? Body is more associated to API requests content, actions is more meaningful for test scenarios.

Reply:
Renamed. I wasn't completely sure with `allure.Body` either.